### ISSUE TITLE
chrome issue with tables

### ISSUE THREAD

**Issue by igorkononenko** [tinymce/tinymce#2846]
1. happens in Chrome only
2. insert 1 x 1 table 
3. type some text inside the table cell, let's say 'aaa'
4. start deleting text with Delete or Backspace buttons
5. when you delete the last character inside the table cell - table gets deleted
   OR
6. select all text inside table cell, paste some text, table disappears

**Comment by nori-h** [comment]
I believe this is a duplicate to #2167. Could you please have a look and let us know if not? Thanks, Nori @ TinyMCE

**Comment by igorkononenko** [comment]
Yes, it is indeed a duplicate of #2167... sorry i didn't know that such
issue already exists

**Comment by tinydylan** [comment]
Hi,

I tested the two scenarios above with TinyMCE 5.2.1 on this [demo page](https://www.tiny.cloud/docs/demo/full-featured/) using Chrome 81 on macOS 10.15.4.

1. Insert 1x1 table, type text, backspace until text is gone. In this scenario, the table remained, which is correct.
2. Insert 1x1 table, type text, select all text in the cell, then press backspace. The table is deleted, which is incorrect.

I also tested in Firefox 75 - it behaves correctly.

So, I can confirm that the bug is still present.

Thanks!
